docs/sqlserver.md

```diff
@@ -0,0 +1,55 @@
+
+```yaml
+metrics:
+  receivers:
+    sqlserver:
+      type: sqlserver
+      collection_interval: 60s
+  service:
+    pipelines:
+      sqlserver:
+        receivers:
+          - sqlserver
+```
+
+## Metrics
+The Ops Agent collects the following metrics from SQL Server.
+Note: In order for any metrics to be retrieved, the receiver must run locally on the same server as the SQL Server installation.
+
+| Metric                                               | Data Type      | Unit        | Labels      | Description |
+| ---                                                  | ---            | ---         | ---         | ---         | 
+| workload.googleapis.com/sqlserver.user.connection.count | Gauge       | connections |             | Number of users connected to the SQL Server. |
+| workload.googleapis.com/sqlserver.lock.wait_time.avg | Gauge          | ms          |             | Average wait time for all lock requests that had to wait. |
+| workload.googleapis.com/sqlserver.lock.wait.rate     | Gauge          | requests/s  |             | Number of lock requests resulting in a wait. |
+| workload.googleapis.com/sqlserver.batch.request.rate | Gauge          | requests/s  |             | Number of batch requests received by SQL Server. |
+| workload.googleapis.com/sqlserver.batch.sql_compilation.rate | Gauge  | compilations/s |          | Number of SQL compilations needed. |
+| workload.googleapis.com/sqlserver.batch.sql_recompilation.rate | Gauge | compilations/s |         | Number of SQL recompilations needed. |
+| workload.googleapis.com/sqlserver.page.buffer_cache.hit_ratio | Gauge | %           |             | Pages found in the buffer pool without having to read from disk. |
+| workload.googleapis.com/sqlserver.page.life_expectancy | Gauge        | s           |             | Time a page will stay in the buffer pool. |
+| workload.googleapis.com/sqlserver.page.split.rate    | Gauge          | pages/s     |             | Number of pages split as a result of overflowing index pages. |
+| workload.googleapis.com/sqlserver.page.lazy_write.rate | Gauge        | writes/s    |             | Number of lazy writes moving dirty pages to disk. |
+| workload.googleapis.com/sqlserver.page.checkpoint.flush.rate | Gauge  | pages/s     |             | Number of pages flushed by operations requiring dirty pages to be flushed. |
+| workload.googleapis.com/sqlserver.page.operation.rate | Gauge         | operation/s | type        | Number of physical database page operations issued. |
+| workload.googleapis.com/sqlserver.transaction_log.growth.count | Gauge | growths    | database    | Total number of transaction log expansions for a database. |
+| workload.googleapis.com/sqlserver.transaction_log.shrink.count | Gauge | shrinks    | database    | Total number of transaction log shrinks for a database. |
+| workload.googleapis.com/sqlserver.transaction_log.usage | Gauge       | %           | database    | Percent of transaction log space used. |
```

Comment:
`percent_used`?

Reply:
@qingling128 and @jefferbrecht as approvers of the scope doc, what are you opinions?

Reply:
While `percent_used` is clearer IMO, I don't feel strongly about renaming this; there's precedent either way (e.g. `agent.googleapis.com/memory/percent_used` vs. `workload.googleapis.com/activemq.memory.usage`).

I'll defer to @igorpeshansky, so unless there's an objection from the other reviewers then I'm fine with changing it to `percent_used`.

Reply:
I went ahead and just changed this. at this point I think all open issues can be resolved